docs/source/api/pystiche_papers.johnson_alahi_li_2016.rst

```diff
@@ -1,6 +1,30 @@
 ``pystiche_papers.johnson_alahi_li_2016``
 =========================================
 
++--------------------------+-----------------------------------------------------------+
+| Title                    | Perceptual Losses for Real-Time Style Transfer and        |
+|                          |                                                           |
+|                          | Super-Resolution                                          |
++--------------------------+-----------------------------------------------------------+
+| Authors                  | Justin Johnson, Alexandre Alahi, and Fei-Fei Li           |
++--------------------------+-----------------------------------------------------------+
+| Citation                 | :cite:`JAL2016`                                           |
++--------------------------+-----------------------------------------------------------+
+| Reference implementation | |repo|_ / |archive|_                                      |
++--------------------------+-----------------------------------------------------------+
+| Variant                  | Model optimization                                        |
++--------------------------+-----------------------------------------------------------+
+| Content loss             | :class:`~pystiche.ops.FeatureReconstructionOperator`      |
++--------------------------+-----------------------------------------------------------+
+| Style loss               | :class:`~pystiche.ops.GramOperator`                       |
++--------------------------+-----------------------------------------------------------+
```

Comment:
What about the :class:`~pystiche.ops.TotalVariationOperator`?

Reply:
Good catch!